bsp/loongson/ls2kdev/drivers/ata/dwc_ahsata.c

```diff
@@ -995,7 +995,7 @@ static int ahci_init_one(int pdev)
     int rc;
     struct ahci_uc_priv *uc_priv = NULL;
 
-    uc_priv = malloc(sizeof(struct ahci_uc_priv));
+    uc_priv = (struct ahci_uc_priv*)malloc(sizeof(struct ahci_uc_priv));
```

Comment:
突然发现一个问题，麻烦改一下 字母和星号之间要有个空格
```suggestion
    uc_priv = (struct ahci_uc_priv *)malloc(sizeof(struct ahci_uc_priv));
```

Reply:
收到，现在就修改

Reply:
已经完成修改，麻烦查看一下